guides/default-technology-stack.md

```diff
@@ -25,11 +28,13 @@ Ruby was selected as the programming language of choice for digital delivery for
 
 We recommend that teams look at the [boilerplate project](https://github.com/DFE-Digital/govuk-rails-boilerplate) when starting their projects.
 
-### The Node.JS stack
+### The .NET stack
```

Comment:
```suggestion
### The .NET core stack
```

Subtle but very different

Reply:
Good point. Can you capitalise the "c" in "core" in your suggestion :)